graphium/ipu/ipu_losses.py

```diff
@@ -107,6 +107,17 @@ def forward(self, input: Tensor, target: Tensor) -> Tensor:
 
 
 class HybridCELossIPU(HybridCELoss):
+    def __init__(
+        self,
+        n_brackets: int = 5,
```

Comment:
Remove default value for `n_brackets`. Pass it in the config file under the metric

Reply:
This is for the loss_func, is it happy to accept something like this?
```
   loss_fun:
    l1000_vcap: hybrid_ce_ipu
        n_brackets: 5
    l1000_mcf7: hybrid_ce_ipu
        n_brackets: 5
```

Reply:
We need to modify the loss_fun to work like the metrics and take parameters. I thought we did that a while ago, but it seems not.

Reply:
```yaml
l1000_cap:
  name: hybrid_ce_ipu
  n_brackets: 5
```

Reply:
This above works fine, closing this conversation.